### DISCUSSION TITLE
ChannelError again

### DISCUSSION THREAD

**Discussion by kirenida** [librespot-org/librespot#999] [General]
It looks like I'm having the same problem as mentioned in bugs #183 and #280.
I pasted my error logs in raspotify issue https://github.com/dtcooper/raspotify/issues/542

But I don't see anybody else having the same issue this time.

I'm located in Oslo, Norway and the problem started on 2022-05-05, in the afternoon by local time.

I haven't changed anything in my local configuration/system, tried restarting my raspberry, router, but the problem is still here. I tried changing the DNS servers in my piHole

Anybody else in the region having the same problem?

Edit 2022-05-08:
Can confirm this is probably a region-based issue, as playback works as usual when I'm running librespot and Spotify dsktop client on my laptop connected through a VPN to a different country, but doesn't work when I'm not using the VPN.

edit2: found issue #972 , blocked 'apresolve.spotify.com', now playback works again.

**Comment by roderickvd** [comment]
In `new-api` we may find a way to propagate `ChannelErrors` up to `librespot-core`, refresh the access point and fall-back to `ap.spotify.com` if `apresolve.spotify.com` continues to bug.

This sounds simple, but while I've sprinkled reconnection logic here and there, there's still a long way to go.
